packages/react/src/wallet/wallets/deviceWallet.ts

```diff
@@ -0,0 +1,45 @@
+import type { WalletOptions, Wallet } from "@thirdweb-dev/react-core";
+import {
+  createAsyncLocalStorage,
+  ConnectParams,
+  DeviceWalletConnectionArgs,
+  DeviceBrowserWallet as DeviceWalletCore,
+} from "@thirdweb-dev/wallets";
+
+// ---
+
+const deviceWalletStorage = createAsyncLocalStorage("deviceWallet");
```

Comment:
this seems a bit scary, if this (for some reason) runs during SSR it would crash - should we instead initialize this during construction time?